app/helpers/games_helper.rb

```diff
@@ -1,37 +1,33 @@
 module GamesHelper
   # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity, Metrics/LineLength
   def piece_present?(pieces, game, row_pos, col_pos)
+    white_pieces = {
+      'Pawn'   => '&#9817;',
+      'Rook'   => '&#9814;',
+      'Knight' => '&#9816;',
+      'Bishop' => '&#9815;',
+      'Queen'  => '&#9812;',
+      'King' => '&#9813;'
```

Comment:
There was an error in how I set up the board.  `&#9812` is the White King, `&#9813` is the White Queen!  They're reversed here.